frontend/packages/console-shared/src/components/formik-fields/field-types.ts

```diff
@@ -87,6 +87,8 @@ export interface YAMLEditorFieldProps extends FieldProps {
   onChange?: (value: string) => void;
   onSave?: () => void;
   schemaModel?: K8sKind;
+  schemaLabel?: string;
+  schema?: any;
```

Comment:
@sahil143 please avoid `any` types at any and all changes you get. I think this is supposed to be `JSONSchema6` based on the singular usage here of `formSchema` in `frontend/packages/dev-console/src/components/helm/form/HelmInstallUpgradeForm.tsx`... please confirm.

cc @rohitkrai03